src/bokeh/core/has_props.py

```diff
@@ -213,7 +213,7 @@ def __init__(cls, class_name: str, bases: tuple[type, ...], _) -> None:
         redeclared = own_properties.keys() & base_properties.keys()
         if redeclared:
             warn(f"Properties {redeclared!r} in class {cls.__name__} were previously declared on a parent "
-                 "class. It never makes sense to do this. Redundant properties should be deleted here, or on"
+                 "class. It never makes sense to do this. Redundant properties should be deleted here, or on "
```

Comment:
This change seems like a mistake, sine it will cause "on" and "the parent" to be concatenated into "onthe parent"

Reply:
This change is doing the opposite of that